pyaedt/modeler/actors.py

```diff
@@ -182,13 +179,13 @@ def _add_flying(self, app):
 
     @aedt_exception_handler
     def insert(self, app, motion=True):
-        """Insert the bird in AEDT.
+        """Insert the bird in HFSS SBR+.
 
         Parameters
         ----------
-        app: :class:`pyaedt.hfss.Hfss`
+        app: pyaedt.Hfss
```

Comment:
@PipKat Is there a reason to remove the class reference here?

Reply:
@jleonatti In the rendered help, the link for this instance was NOT working, but the link for a reference without the class reference was working. I couldn't figure out why, so I copied the working instance and pasted here to see if it fixed the issue.

Reply:
I reviewed the doc using the two syntaxes and I see what you're saying, so this syntax is fine.